packages/opencensus-web-instrumentation-zone/test/test-interaction-tracker.ts

```diff
@@ -392,6 +437,25 @@ describe('InteractionTracker', () => {
       expect(childSpan.attributes[ATTRIBUTE_HTTP_STATUS_CODE]).toBe('200');
       expect(childSpan.attributes[ATTRIBUTE_HTTP_METHOD]).toBe('GET');
       expect(childSpan.ended).toBeTruthy();
+      const childSpanPerfEntries = getXhrPerfomanceData(urlRequest, childSpan);
+      expect(childSpanPerfEntries).toBeTruthy();
+      if (childSpanPerfEntries instanceof Array) {
```

Comment:
Rather than checking here, can we know what it should be and `expect` it? (I think that applies to a case above too)